proc/proc.go

```diff
@@ -740,37 +732,21 @@ func initializeDebugProcess(dbp *Process, path string, attach bool) (*Process, e
 
 func (dbp *Process) ClearTempBreakpoints() error {
 	for _, bp := range dbp.Breakpoints {
-		if !bp.Temp {
+		if bp.Kind == NormalBreakpoint {
```

Comment:
Maybe we add a method to `Breakpoint` such as `Temp() bool` which encapsulates this logic? Just a minor nit.